src/components/ReleaseBanner.tsx

```diff
@@ -57,6 +25,42 @@ interface CountdownRendererProps {
   completed: boolean
 }
 
+interface BannerWrapperProps {
+  isBannerVisible: boolean
+  onClose: () => void
+  children: JSX.Element
+}
+
+const BannerWrapper: React.FC<BannerWrapperProps> = ({
+  onClose,
+  isBannerVisible,
+  children,
+}) => {
+  return (
+    <BannerNotification shouldShow={isBannerVisible} position="relative">
+      <Center position="relative" paddingRight="2rem" flexWrap="wrap">
+        <Emoji
+          marginRight="1rem"
+          flexShrink={0}
+          text=":tada:"
+          w="1rem"
+          h="1rem"
+        />
+        {children}
+      </Center>
+      <IconButton
+        aria-label="Close Banner"
+        icon={<CloseIcon />}
```

Comment:
We should not use the Chakra icons, so lets revert that part. What the migration guide says is to use the Chakra `Icon` component (or in this case the `IconButton` component).

We should still use the same svgs we were using before. In this case we were using `MdClose` which you can import it by doing `import { MdClose } from "react-icons/md"`

And finally, you can do `<IconButton icon={<MdClose />}`

Reply:
Got it
Removed `@chakra-ui/icons` and used `<MdClose />`